README.md

```diff
@@ -80,6 +80,27 @@ client.users.create({ email: 'foo@bar.com' }).then(function (r) {
 });
 ```
 
+## Request Options
+
+This client library also supports passing in [`request` options](https://github.com/request/request#requestoptions-callback):
+
+```node
+var client = new Intercom.Client({ token: 'my_token' });
+client.useRequestOpts({
+  // Uses the forever-agent / http(s).Agent({keepAlive:true})
+  forever: true
+});
+```
+
+Note that certain request options (such as `json`, and certain `headers` cannot be overriden).
+
+To change the `baseUrl` for the client library use:
+
+```node
+var client = new Intercom.Client({ token: 'my_token' });
+client.useBaseUrl('http://local.test-server.com');
+```
```

Comment:
@mattdonnelly after updating the README to include the `.useBaseUrl()` method, would it be worth deprecating that (with the functionality still working but with a warning) in place of using `.useRequestOpts({ baseUrl: 'blah' })`?

Reply:
Good idea, I think we can remove it from the README, but we should keep the `useBaseUrl` method until we decide to make a major version release

Reply:
Done!